package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "cra-template-redux",
+  "version": "1.0.0",
+  "scripts": {
+    "prettify": "prettier './template/src/**/*.js' --write",
```

Comment:
Maybe we should use CRA's Prettier config so there's less noise if someone switches from the default template.

Reply:
They don't have a config file but I added the same flags they use here: https://github.com/facebook/create-react-app/blob/master/package.json#L19